### DISCUSSION TITLE
How do I had symball 208.1

### DISCUSSION THREAD

**Discussion by tomasweiss** [OpenOrienteering/mapper#2031] [Questions & Answers]
Hi, i want to map a line of rocks along a path- to many to map individually, so, I want to map it as "bolder field"  (208) , but since It's a line , I need to do so using few single triangle symballs - but I can't find it in the symball set , is there a way to upload such symball?

**Comment by mlerjen** [comment]
208.1 is available in both the Mapper ISSprOM and ISOM Symbolset.
![image](https://user-images.githubusercontent.com/15611653/151546583-dde7c697-953b-4bb2-a040-1c870a6c4fa0.png)

**Comment by dl3sdo** [comment]
If it's just about distributing rocks along a path, you could do the following:
- use an existing line object or draw an auxiliary line object
- select that line
- select 208.1 for example
- use the 'Distribute points along path' tool

**Reply by tomasweiss** [reply]
Ok , got it , thanks
